### ISSUE TITLE
A code error

### ISSUE THREAD

**Issue by yang-ze-kang** [DearCaat/MHIM-MIL#23]
I see an error in your code. When applying gradient accumulation, the 'train_loss.backward()' should be applied in each batch.
